webrtc/src/track/track_local/track_local_static_sample.rs

```diff
@@ -52,6 +55,40 @@ impl TrackLocalStaticSample {
             return Ok(());
         }
 
+        let (any_paused, all_paused) = (
+            self.rtp_track.any_binding_paused().await,
+            self.rtp_track.all_binding_paused().await,
+        );
+
+        if all_paused {
+            // Abort already here to not increment sequence numbers.
+            return Ok(());
+        }
+
+        if any_paused {
+            // This is a problem state due to how this impl is structured. The sequencer will allocate
+            // one sequence number per RTP packet regardless of how many TrackBinding that will send
+            // the packet. I.e. we get the same sequence number per multiple SSRC, which is not good
+            // for SRTP, but that's how it works.
+            //
+            // Chrome has further a problem with regards to jumps in sequence number. Consider this:
+            //
+            // 1. Create track local
+            // 2. Bind track local to track 1.
+            // 3. Bind track local to track 2.
+            // 4. Pause track 1.
+            // 5. Keep sending...
+            //
+            // At this point, the track local will keep incrementing the sequence number, because we have
+            // one binding that is still active. However Chrome can only accept a relatively small jump
```

Comment:
Ah ok. That makes sense. The ROC will only be correct if we've had a gradual increase on sequence numbers without big jumps. That's actually an implementation agnostic problem.

Reply:
Yes exactly, if you pause before a rollover and then unpause after it all proper SRTP implementations should reject all subsequent RTP packets because of the auth tag check failing